doc/api/videoThumbnailLoader.md

```diff
@@ -0,0 +1,118 @@
+# VideoThumbnailLoader #########################################################
+
+
+## Overview ####################################################################
+
+The VideoThumbnailLoader is a tool that can help using a video track
+(Representation) as a substitue of a manifest embedded thumbnail track.
+
+The goal is to make a thumbnail out of HTML5 video element, by :
+- Managing the loading / appending of resources from a given track
+(video segments).
+- Exploiting the Media Source Extension API to make it invisible to user.
+
+## How to use it ###############################################################
+
+
+As an experimental tool, the VideoThumbnailLoader won't be included in a
+default RxPlayer build.
+
+Instead, it should be imported by adding the RxPlayer through a dependency
+trough the npm registry (e.g. by doing something like ``npm install
+rx-player``) and then specifically importing this tool from
+``"rx-player/experimental/tools"``:
+
+```js
+import { VideoThumbnailLoader } from "rx-player/experimental/tools";
+
+const currentAdaptations = player.currentAdaptations();
+if (
+  currentAdaptations.video &&
+  currentAdaptations.video.trickModeTrack &&
+  currentAdaptations.video.trickModeTrack.representations[0]
+) {
+  const track = currentAdaptations.video.trickModeTrack.representations[0];
+  const videoElement = document.createElement("video");
+  const videoThumbnailLoader = new VideoThumbnailLoader(
+    videoElement,
+    track
+  );
+}
```

Comment:
That's a complicated API.
Also what about multi-period contents?

Maybe we should just give the Manifest object and that's it.
That's a compromise